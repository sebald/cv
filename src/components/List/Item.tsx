import { styled } from '../../style';

export const Item = styled.li``;
