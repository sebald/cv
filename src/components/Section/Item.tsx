import * as React from 'react';

import { padding, styled } from '../../style';
import { Caption, Subheading } from '../Typography';
import { SectionItemProps } from './types';

const ItemWrapper = styled.div`
  ${padding(0, 0, 3)};
`;

export const Item: React.FC<SectionItemProps> = ({
  title,
  caption,
  children,
}) => (
  <ItemWrapper>
    <Subheading>{title}</Subheading>
    {caption && <Caption>{caption}</Caption>}
    {children}
  </ItemWrapper>
);
