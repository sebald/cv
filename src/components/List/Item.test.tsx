import { shallow } from 'enzyme';
import * as React from 'react';

import { Item } from './Item';

test('render correct', () => {
  const wrapper = shallow(<Item />);
  expect(wrapper).toMatchSnapshot();
});
