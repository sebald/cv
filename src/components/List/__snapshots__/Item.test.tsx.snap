// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correct 1`] = `
<li
  className=""
/>
`;
